vllm/v1/engine/async_llm.py

```diff
@@ -636,9 +636,25 @@ async def sleep(self, level: int = 1) -> None:
         await self.reset_prefix_cache()
         await self.engine_core.sleep_async(level)
 
+        # Log sleep status
+        if self.log_stats and self.logger_manager:
+            model_name = self.vllm_config.model_config.served_model_name
+            for engine_idx in self.logger_manager.engine_idxs:
+                engine_stats = EngineStateStats(sleep=1, level=level)
```

Comment:
There is no need for this `EngineStateStats` object - just pass sleep and level parameters

Reply:
Agreed! I just removed `EngineStateStats` object.